variables.tf

```diff
@@ -41,7 +41,7 @@ variable "cidr_blocks" {
 variable "subnet_role" {
   description = "A role used to filter out which subnets the RDS should reside, defaults to Role=compute"
   default     = "compute"
-}
+ }
```

Comment:
Run a `terraform fmt` within the directory, which will probably correct this bit